server/.env.example

```diff
@@ -1,7 +1,7 @@
 # General
 PORT=8080
 REEARTH_DB=mongodb://localhost     # for reearth database url
-REEARTH_DB_ACCOUNT=reearth_account # for reearth account database name
+REEARTH_DB_ACCOUNT=reearth-account # for reearth account database name
```

Comment:
[nits]
Could you leave this as it is? Changing it would affect those who have been using it.